pkg/volumereplica/volumereplica.go

```diff
@@ -126,36 +128,36 @@ func PoolNameFromCVR(cvr *cstor.CStorVolumeReplica) string {
 
 // ToDo: Move this to backup package
 
-//// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
-//// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
-//// env
-//func PoolNameFromBackup(bkp *cstor.CStorBackup) string {
-//	poolname := bkp.Labels[CStorPoolUIDKey]
-//	if strings.TrimSpace(poolname) == "" {
-//		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
-//		if strings.TrimSpace(poolname) == "" {
-//			return ""
-//		}
-//	}
-//	return PoolPrefix + poolname
-//
-//}
+// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
+// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
+// env
+func PoolNameFromBackup(bkp *openebsio.CStorBackup) string {
+	poolname := bkp.Labels[CStorPoolUIDKey]
```

Comment:
labels need to be changed according to new labels. Similarly in all other places.

Reply:
https://github.com/openebs/cstor-operators/pull/83#issue-423930864
The above description has restore labels.